docs/content/getting-started.md

```diff
@@ -51,37 +50,33 @@ Let's start with the contents of `index.html`:
   <div router-view-port></div>
 
   <script src="/node_modules/angular/angular.js"></script>
-  <script src="/dist/router.es5.js"></script>
-  <script src="/app/app.js"></script>
+  <script src="/node_modules/angular-new-router/dist/router.es5.js"></script>
+  <script src="/app.js"></script>
 </body>
 </html>
 ```
 
 This is a pretty typical angular app, except the `router-view-port` directive.
-`router-view-port` is like `ng-view`; it's a placeholder for part of your app loaded
-dynamically based on the route configuration.
+`router-view-port` is like `ng-view`; it's a placeholder for part of your app loaded dynamically based on the route configuration.
 
 So how do we configure the app? Let's open `app.js` and find out. Add this to the file:
 
 ```js
-angular.module('app', ['ngNewRouter'])
+angular.module('myApp', ['ngNewRouter'])
```

Comment:
We should decide here if we want to use `app` or `myApp`. This PR introduces `myApp` but still uses `app` later on